test/app.spec.js

```diff
@@ -430,7 +457,7 @@ describe('app', () => {
     it('sets source for auth messages', () => {
       const authMessage = Object.assign({optin: 'foo'}, baseMessage);
       return messenger.routeEachMessage(authMessage)
-        .then(() => app.__internals__.cache.get(messenger.getCacheKey(baseMessage.sender.id)))
```

Comment:
I've actually been thinking about putting the cache inside `Messenger` for awhile to get rid of this awkwardness.